src/base/TilesRendererBase.js

```diff
@@ -716,7 +719,7 @@ export class TilesRendererBase {
 						const uri = parseTile.content.uri;
 						const extension = getUrlExtension( uri );
 
-						return this.parseTile( buffer, parseTile, extension );
+						return this.invokeOnePlugin( plugin => plugin.parseTile && plugin.parseTile( buffer, parseTile, extension ));
```

Comment:
We need to make sure that any child tiles created and added to the tile set are disposed of correctly when this tile is evicted from the cache. Effectively any members and child tiles added to the tile when "parsing" it should be disposed so when this tile is loaded again we can re load and regenerate the implicit children.

Reply:
Looks to me that it got disposed as any "classic" children.
Only the .subtree files will be loaded from the plugin, they don't have geometry or anything. Do you think will be issues with this ?

Reply:
What do you mean by "classic" children? As in children with geometry?

When a tile is no longer meeting the error target or in the frustum it's queued up for disposal if it has any kind of content that needed to be loaded. This includes [external tile sets](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/7f95332ea3f31a97c1b226e3ea38b8ed720183ab/src/base/TilesRendererBase.js#L538-L542). The children that are loaded (or generated in this case) still take up CPU memory and with enough loaded that could be substantial.

For now the subtree children are only being expanded for the root tile but those children should be disposed of to be consistent with the existing behavior of the TilesRenderer and will be more necessary once deeper child subtrees are supported.

Reply:
Could it be enough to check it via the flag `__hasRenderableContent`, WDYT? As following:
```js
} else if ( isExternalTileSet || !t.__hasRenderableContent) {
    t.children.length = 0;
}
```

Reply:
I think I'd prefer the plugin be responsible for cleaning up at after itself since it adds the children instead of having the creation and clean up happen in two separate places.

Reply:
You mean like this : 
```js
	if ( t.__loadingState === LOADING ) {

		t.__loadAbort.abort();
		t.__loadAbort = null;

	} else if ( isExternalTileSet ) {

		t.children.length = 0;

	} else {

		this.invokeAllPlugins( plugin => {

			plugin.disposeTile && plugin.disposeTile(t);

		} );
	}
```

With inside `ImplicitTilingPlugin.js` : 

```js
disposeTile(tile){
  if (/.subtree$/i.test(tile.content?.uri)){
	  tile.children.length = 0;
  }
}
```

Reply:
Yup! Like that.